src/View/ViewVarsTrait.php

```diff
@@ -38,6 +38,7 @@
      * Variables for the view
      *
      * @var array
+     * @deprecated 3.7.0 Use `$this->viewBuilder()->setVar()/getVar()` instead.
```

Comment:
Couldn't folks also use `$this->set()` for a simpler interface?

Reply:
Yes they can, but currently if you use `$this->set()` you will have to still use `$this->viewVars` if you want to read the vars.

Reply:
My concern would be that people migrate to using the longer form and think that 'cake is too verbose for simple things'. This message could be:

> Use `$this->set()` to set variables or `$this->viewBuilder()->getVar()` to read them instead.`

Reply:
The thing is if you use `$this->set()` you can't `$this->viewBuilder()->getVar()` to read the var as `$this->set()` still sets them to `$this->viewVars` not to the viewbuilder.

Reply:
Hrm.

Reply:
Done